src/main/java/controllers/WorkoutStartController.java

```diff
@@ -0,0 +1,194 @@
+package controllers;
+
+import management.RoutineManager;
+import management.Workout;
+import views.WorkoutMenuView;
+import views.WorkoutSelectionView;
+import views.WorkoutStartView;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class WorkoutStartController {
+    private final RoutineManager startModel;
+    private final WorkoutStartView startView;
+
+    public WorkoutStartController(RoutineManager startModel, WorkoutStartView startView) {
+
+        this.startModel = startModel;
+
+        this.startView = startView;
+
+        this.startView.nextExerciseDisplay(new NextStartDisplay());
+
+    }
+
+    /**
+     * Implements the behaviour of the next button.
+     */
+    class NextStartDisplay implements ActionListener {
+
+        @Override
+        public void actionPerformed(ActionEvent e) {
```

Comment:
Code smell in the form of a bloater. The method body is extremely huge. Maybe try extracting some of the if statements into another method, or find another work around. The code is hard to follow because of how big it is.